src/viam/app/app_client.py

```diff
@@ -444,6 +479,10 @@ async def create_organization(self, name: str) -> Organization:
     async def list_organizations(self) -> List[Organization]:
         """List the organization(s) the user is an authorized owner of.
 
+        ::
```

Comment:
(q) I'm not sure what this is doing exactly. Clearly it's intentional since the pattern appears repeatedly, does the `::` interact with doc generation in some way?

Reply:
it makes the code snippet appear as code in the generated docs
See here for example: https://python.viam.dev/autoapi/viam/components/gripper/index.html

Reply:
Cool thanks for confirming!